### PR TITLE
Create Leveraging Metadata Collections Guide.

### PR DESCRIPTION
**Pull request by markpbaggett**
# What Does This Do

Creates a Guide for Leveraging Generated Collections in Custom Content.

# Notes

While cooking, I feel like I may have over seasoned this. Hopefully, I can salvage it before dinner. 🍽️

### CODE REVIEW

File: pages/leverage-metadata-collections.mdx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+<Callout>
+  This guide assumes you have a Canopy IIIF project. See the [Create a
+  Project](/get-started) guide to get started.
+</Callout>
+
+## Use Case
+
+You are a researcher creating a digital scholarship site and would like to create a custom content page
+with a Slider component that displays all the works in your collection that have a "Subject" of "War pensions".
+
```
Comment (adamjarling) [inline pages/leverage-metadata-collections.mdx:25]: Maybe unnecessary, but what if there was a quick little snippet of `json` here illustrating where "Subject" sits within the example Manifest's `metadata`?

**Review by adamjarling — APPROVED** [review]
This is really good @markpbaggett A few quick small ideas/suggestions, but take 'em or leave 'em.

File: pages/leverage-metadata-collections.mdx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+# War Pensions
+
+During the American Civil War, the issue of war pensions emerged as a crucial aspect of post-conflict reconstruction. As
+the nation grappled with the devastating aftermath of the war, the U.S. government recognized the need to support the
+veterans who had sacrificed their well-being for the Union or the Confederacy. War pensions, providing financial
+assistance and support to disabled veterans, widows, and orphans, became a cornerstone of the post-war social contract.
+The Pension Acts of 1862 and 1864 marked significant legislative milestones, establishing the framework for compensating
+soldiers and their families for injuries, disabilities, and loss of life incurred during the conflict. These early
+pension initiatives laid the foundation for the more comprehensive pension system that would evolve in subsequent
+decades, reflecting the government's commitment to honoring and caring for those who bore the burdens of the Civil War.
```
[the remark this answers is not in the document]
Comment (markpbaggett) [reply pages/leverage-metadata-collections.mdx:120]: You can thank ChatGPT for that.

File: pages/leverage-metadata-collections.mdx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+## Explore Works Related to War Pensions
+
+<Slider iiifContent="/api/facet/subject/war-pensions?sort=random" />
```
[the remark this answers is not in the document]
Comment (markpbaggett) [reply pages/leverage-metadata-collections.mdx:161]: Nope, drop it from the guide and revisit in the future:

https://github.com/canopy-iiif/canopy-iiif/issues/172

**Comment by markpbaggett** [comment]
@mathewjordan Thanks for the feedback and catching the static problem.  As a result, I've committed each of your suggestions, dropped step 4, and caught a few other non `.json` routes that weren't caught.  

Let me know if you notice anything else.

**Review by mathewjordan — APPROVED** [review]

**Pull request merged**